lib/vagrant-openshift/action/install_origin_base_dependencies.rb

```diff
@@ -215,6 +215,12 @@ def call(env)
   sudo lvcreate -n docker-data -l 90%FREE /dev/${VG}
   sudo lvcreate -n docker-metadata -l 50%FREE /dev/${VG}
   sudo sed -i "s,^DOCKER_STORAGE_OPTIONS=.*,DOCKER_STORAGE_OPTIONS='-s devicemapper --storage-opt dm.datadev=/dev/${VG}/docker-data --storage-opt dm.metadatadev=/dev/${VG}/docker-metadata'," /etc/sysconfig/docker-storage
+
+  sudo lvcreate -n openshift-xfs-tmp -l 100%FREE /dev/${VG}
+  sudo mkfs.xfs /dev/${VG}/openshift-xfs-tmp
+  sudo mkdir -p /tmp/openshift
```

Comment:
In fairness, I probably could do this with a new dir, I do have to modify node-config for the test anyhow so it wouldn't be hard to point volumeDirectory at something else.

This is getting a bit more coupled, the test would have to assume a LV or a mount point exists, but it could definitely be done. 

It would also be nice to be able to reformat the volume after or before the test runs, as there is no reasonable way to delete an XFS quota from the filesystem after it's applied.

I have the tests running now, just a few minutes in and it's at 386M so indeed 1G is probably going to fail.

I'll consider this a sign to go with a separate volume directory for this particular test. I'll mount to /tmp/openshift/xfs-volume-dir. Would it be safe to trash it and format the fs each test? (I.e. assuming that it's safe to mess with an LV with this name) Getting pretty tightly coupled but I don't see any other way.